### DISCUSSION TITLE
How to bypass admin blocked to do HSTS on iboss.

### DISCUSSION THREAD

**Discussion by IGETboard** [3kh0/ext-remover#1402] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
How to bypass admin blocked to do HSTS on iboss.

**Comment by Luigi-Master** [comment]
I don't know that exploit but there is another way to bypass iBoss but I don't know if its patched, enable "Extension Manifest V2 Deprecation Warning Stage" in chrome flags, If it dosen't work, its patched.

**Reply by IGETboard** [reply]
I cant find that flag, so i am guessing as of now no new bypasses are avalible.

**Reply by RacoonyCoon** [reply]
https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/Disable%20Extensions%20-%20HSTS
I use this and it works perfectly, I have to use a VPN however when at school because my school wifi is managed

**Reply by IGETboard** [reply]
I used to use that until it got blocked and i am trying to find another way.

**Reply by RacoonyCoon** [reply]
I don't think it can be blocked if it stopped working you might of cleared your cache and history or something like that I'd reccomend to try it again and see if it works. unless you mean chrome net-internals got blocked

**Reply by IGETboard** [reply]
Yeah net-internals is blocked.

**Comment by S-PScripts** [comment, marked as answer]
yearlong expiry
